src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -0,0 +1,213 @@
+<div style="padding-bottom: 60px">
+    <h2 style="padding-bottom: 20px">
+        <span id="build-queue-page-heading" jhiTranslate="artemisApp.buildQueue.pageTitle"> Build job queue information </span>
+        <button class="btn btn-primary float-end jh-create-entity" (click)="ngOnInit()">
```

Comment:
I would rather not call `ngOnInit` here. 
What do you think about putting all the service calls in the component into a load() function and call this one here and in ngOnInit?